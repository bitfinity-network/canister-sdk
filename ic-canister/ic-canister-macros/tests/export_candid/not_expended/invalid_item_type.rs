fn main() {}

#[ic_canister_macros::export_candid]
struct Did {}
