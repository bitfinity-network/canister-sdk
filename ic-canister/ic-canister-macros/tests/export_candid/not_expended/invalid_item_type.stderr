error: expected `fn`
 --> tests/export_candid/not_expended/invalid_item_type.rs:4:1
  |
4 | struct Did {}
  | ^^^^^^
